spec/views/hq/orphans/show_spec.rb

```diff
@@ -11,7 +11,9 @@
       expect(rendered).to match @orphan.name
```

Comment:
No what I mean is the gem or wherever it's coming from using `CGI::escape_html` or is it using some other html escaping method that maybe handles certain characters differently?  I remember at least one test somewhere that wouldn't pass without this.  Or could it be the Minitest assertions working differently than `expect(rendered).to match...`